howtos/how_to_request_cached_data.py

```diff
@@ -0,0 +1,139 @@
+"""
+This module contains example snippets on how to request cached data from a direct receiver. These examples assume
+that the direct receiver has already been started, and that it was created from a messaging service which has
+already been connected.
+"""
+
+from solace.messaging.utils.correlation_completion_listener import CorrelationCompletionListener
+from solace.messaging.utils.cache_request_outcome import CacheRequestOutcome
+from solace.messaging.resources.topic_subscription import TopicSubscription
+from solace.messaging.resources.cached_message_subscription_request import CachedMessageSubscriptionRequest
+
+class MyCorrelationCompletionListener(CorrelationCompletionListener):
+    def on_completion(result: CacheRequestOutcome, correlation_id: int, e: Exception):
+        print("Completed!")
+
+class HowToUseCachedMessageSubscriptionRequests:
+    """
+    Sampler for making cache requests. Usage of cache subscriptions require
+    configuration and hosting of SolCache application.
```

Comment:
Use the name PubSub+ Cache now, not SolCache

Reply:
Please review fix

Reply:
If you can have the + character without messing up some docs, it's usually shown that way instead of spelling out Plus (Eg. https://docs.solace.com/Additional-Products/Solace-PubSub-Cache/PubSub-Cache-Overview.htm)

Reply:
Please review fix.

Reply:
LGTM